web-hosting-case-study.md

```diff
@@ -0,0 +1,372 @@
+
+- cPanel
+
+- One-Click Installer
+
+- File Transfer Protocol
+
+- E-Commerce Features
+
+### Domain Names
+
+Your domain name is the digital address of your digital space on the internet. The reason behind making this was to find and identify internet entities, but without using IP addresses. A domain name is basically an Internet resource name that is universally understood by Web servers and online organizations and provides all pertinent destination information. This makes everything more safe and organized.
+
+### Website Builder
+
+Website builder is a tool or program that can help you create a website without actual manual code editing. There is two times of website builders:
+
+- Offline Website Builders
+
+- Online Website Builders
+
+### Support
+
+Most of the times you can never create a website 100% error-free, there will always bee some errors down the road which you will have to figure out how to fix. In those cases, a technical support team is something you will need. The biggest and best web hosting providers provide 24/7 customer support in many different ways, these can be such as:
+
+- Knowledge bases
+
+- Forums
+
+- Live chat
+
+- Chatbots
+
+- Email support
+
+- Calls
+
+### Backups 
+
+Backups is very important. It can prevent data loss, which is very important. It does not matter how good your website is, a website can always get hit by something which will take the website down. A good web hosting company should restore your website with minimal downtime and make frequent automatic backups of your website to like I said prevent data loss.
+
+## Strengths
+
+![strengths-image](http://nejcslovnik.com/wp-content/uploads/2016/09/Image_web.jpg)
+
+### Shared Hosting
+
+- Low cost: one of the strengths with web hosting is low cost, , thats one of the main advantages.
+
+- Easy to use: it`s easy to use web hosting where it is simple to install and maintain the site trough a system where you just "click and drag" when facilitating the process.
+
+- Technical Support: the good thing is that the technical support is guaranteed by most of the  companies that offer web hosting, so it`s not scary or a risk to do it if you are not sure. Here the user will get help with important factors, such as the website itself, the server and configuration requirements.
+
+- Resources: another advance is resources, here you can get a collection of resources that can grow and get bigger while following the websites requirements and demands.
+
+### VPS Hosting (Virtual Private Server)
+
+- One of the strengths with VPS is that it works better than shared hosting because it has improved performance and security, something shared hosting don`t have.
+
+- It`s cost benefit, it saves you money.
+
+- Get the acquisition and maintenance of the equipment, and you will get reduction in the cost.
+
+### Dedicated Server Hosting
+
+- The biggest strength is that it`s very safe, zero risk of being harmed by other companies.
```

Comment:
```suggestion
- The biggest strength is that it's very safe, zero risk of being harmed by other companies.
```

This is quite a bold statement... Do you have a source for it?

Reply:
Fixed the grammar. And answered the answer to this question in the file.